src/components/navigation-header/components/dropdown-menu/index.tsx

```diff
@@ -1,13 +1,19 @@
+// Third-party imports
 import { KeyboardEvent, ReactElement, useMemo, useRef, useState } from 'react'
 import Link from 'next/link'
 import slugify from 'slugify'
+
+// HashiCorp imports
 import { IconChevronDown16 } from '@hashicorp/flight-icons/svg-react/chevron-down-16'
 import { IconDocs16 } from '@hashicorp/flight-icons/svg-react/docs-16'
 import { IconHome16 } from '@hashicorp/flight-icons/svg-react/home-16'
 import { IconTerminalScreen16 } from '@hashicorp/flight-icons/svg-react/terminal-screen-16'
 import { IconTools16 } from '@hashicorp/flight-icons/svg-react/tools-16'
+
+// Global imports
```

Comment:
The import list was getting long, so I added these comments to try to break it up a little bit. Not looking for a standard practice or anything, and can remove if it interferes with anything else!